docs/proposals/policy.md

```diff
@@ -0,0 +1,146 @@
+
+
+Policy extends the existing authorization rules in Kubernetes to fulfill requirements around extended verb sets, verb specific attributes, role based management of attribute based access control, and more efficient evaluation of authorization status at larger scales.
+
+
+### Runtime Authorization Types
```

Comment:
These types belong later in the file - they are implementation detail, not clarification about why this needs to be done and use case driven.